src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -247,12 +247,10 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                         switch (url)
                         {
                             case var _ when url == "":
-                            case var _ when url.StartsWith("wasm://", StringComparison.Ordinal):
-                            case var _ when url.EndsWith(".wasm", StringComparison.Ordinal):
```

Comment:
Does this have any negative interaction with the code that filters unmanaged frames out of the callstack?

Reply:
As far as I saw no. I will check it again.

Reply:
With this PR:
![image](https://user-images.githubusercontent.com/4503299/170550967-f86bee3d-68f0-4528-95ec-4451da7ed601.png)
Without this PR:
![image](https://user-images.githubusercontent.com/4503299/170551266-606f3719-0504-4337-8997-e5e4c56ebb90.png)